libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html

```diff
@@ -23,12 +31,16 @@
         >&nbsp;</i
       >
     </div>
-    <!-- Non-breaking space needed to align switch to baseline! -->
-    <span *ngIf="hasLabelComponent" class="sky-toggle-switch-label">
-      <ng-container *ngTemplateOutlet="labelContent"></ng-container>
-    </span></button></span
-><span class="sky-control-help-container"
-  ><ng-content select=".sky-control-help"></ng-content
+  </button>
+  <span
```

Comment:
Do we still need this `span` element?

Reply:
That is what we're using to maintain the spacing that UX wants.